Makefile

```diff
@@ -158,23 +167,28 @@ dev3-clean: internal-clean
 dev3-distclean: KIND=dev3
 dev3-distclean: internal-distclean
 
-dialyzer-install:
-	@./rebar3 tree
-	@./rebar3 dialyzer -u true -s false
+dialyzer-install: $(SWAGGER_ENDPOINTS_SPEC)
+	@$(REBAR) tree
+	@$(REBAR) dialyzer -u true -s false
 
-dialyzer:
-	@./rebar3 dialyzer
+dialyzer: $(SWAGGER_ENDPOINTS_SPEC)
+	@$(REBAR) dialyzer
 
-test:
+ct: KIND=test
+ct: internal-build
 	@EPOCH_PROCESSES="$$(ps -fea | grep bin/epoch | grep -v grep)"; \
 	if [ $$(printf "%b" "$${EPOCH_PROCESSES}" | wc -l) -gt 0 ] ; then \
 		(printf "%b\n%b\n" "Failed testing: another Epoch node is already running" "$${EPOCH_PROCESSES}" >&2; exit 1);\
 	else \
-		./rebar3 as test do release, ct $(CT_TEST_FLAGS) --sys_config config/test.config; \
+		$(REBAR) ct $(CT_TEST_FLAGS) --sys_config config/test.config; \
 	fi
 
-eunit:
-	@ERL_FLAGS="-args_file $(EUNIT_VM_ARGS)" ./rebar3 do eunit $(EUNIT_TEST_FLAGS)
+REVISION:
```

Comment:
this won't work once you change the HEAD

Reply:
I mean, it will keep the old REVISION file

Reply:
Its `phony`, so it should be re-executed all the time.